tests/k3s/install-k3s.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Install k3s
+# Since the latest k3s already bakes in the Spin shim, we don't need to reconfigure it.
+# We will just need to make sure that containerd-shim-spin-v2 binary is in PATH
+# and that the k3s service is running.
+
+curl -sfL https://get.k3s.io | sh -
+sudo systemctl start k3s
+sudo k3s kubectl get nodes
+
+sudo chmod 644 /etc/rancher/k3s/k3s.yaml
+sudo cp /etc/rancher/k3s/k3s.yaml $HOME/.kube/k3s.yaml
+export KUBECONFIG=$HOME/.kube/k3s.yaml
```

Comment:
This setting wont live outside this script. We likely need to set that before executing the tests in whatever shell does that or chang ethis to `cp /etc/rancher/k3s/k3s.yaml $HOME/.kube/config`

Reply:
Ah that's a good point